zerver/views/integrations.py

```diff
@@ -19,14 +19,21 @@
 from zerver.lib.zapier_intergrations import ZAPIER_INTEGRATIONS
 from zerver.lib.ifttt_integrations import IFTTT_INTEGRATIONS
 
-# add Zapier integrations into the dict
-for zapier_integration in ZAPIER_INTEGRATIONS:
-    INTEGRATIONS[zapier_integration.name] = zapier_integration
+
+# This is bit of hack to give preference to integrations in
+# the order- Zapier, IFTTT, Hubot.
+# In the dictionary first Hubot is added then IFTTT and
+# then Zapier so that the same named apps are in the preferred
+# order.
 
 # add IFTTT integrations into the dict
 for ifttt_integration in IFTTT_INTEGRATIONS:
     INTEGRATIONS[ifttt_integration.name] = ifttt_integration
 
+# add Zapier integrations into the dict
```

Comment:
in general I wouldn't add comments like this into the code (i.e. comments that explain something that one can see by reading the code itself), since the comments can go out of date